docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,306 @@
+    "</br></br>\n",
+    "<u>Serving creation</u>:</br>\n",
+    "First, we will create our model file and save it as a .py file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# demo, saved in /your/path/add_ten_model.py\n",
+    "from mlrun.serving import V2ModelServer\n",
+    "\n",
+    "class AddTenModel(V2ModelServer):\n",
+    "    def load(self):\n",
+    "        # No actual model to load, just a demo\n",
+    "        pass\n",
+    "\n",
+    "    def predict(self, request):\n",
+    "        input = request['inputs'][0]\n",
+    "        result = input + 10\n",
+    "        return {\"outputs\": [result]}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "Now we will deploy this model as a serving function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# first we will create a demo model, using V2ModelServer, we will save this file as .py file.\n",
+    "\n",
+    "# Import MLRun and create the serving function\n",
+    "import mlrun\n",
+    "project_name = \"serving-project\"\n",
+    "model_name = \"add-ten-model\"\n",
+    "model_path = \"/your/path/add_ten_model.py\"\n",
+    "\n",
+    "project = mlrun.get_or_create_project(project_name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create the serving function\n",
+    "function = mlrun.code_to_function(name='add-ten-serving',\n",
+    "                                  kind='serving',\n",
+    "                                  image='your-image', # should include mlrun for importing V2ModelServer\n",
+    "                                  filename=model_path)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Add the model to the function (even though there's no real model in this case)\n",
+    "function.add_model(model_name, model_path=model_path,class_name='AddTenModel')\n",
+    "function.deploy()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# An example of invoke:\n",
+    "function.invoke(f'/v2/models/{model_name}/infer', {\"inputs\":[20]})['outputs']['outputs']"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "<u>Application creation</u>:</br>\n",
+    "At the begining, ee will create our flask server application.</br>\n",
+    "The application will include several different endpoints in order to check its functionality:"
```

Comment:
"First, create the flask server application.</br>\n",
    "The application includes several different endpoints - to check its functionality:"